errors/manifest.json

```diff
@@ -423,10 +423,6 @@
           "title": "static-page-generation-timeout",
           "path": "/errors/static-page-generation-timeout.md"
         },
-        {
-          "title": "page-data-collection-timeout",
-          "path": "/errors/page-data-collection-timeout.md"
-        },
```

Comment:
I think we need to leave this in the manifest and point it to the new path so it doesn't 404 for previous verisons